packages/cli-e2e/__tests__/atomic.specs.ts

```diff
@@ -51,6 +51,42 @@ describe('ui:create:atomic', () => {
   const getProjectName = (id: string) =>
     `${process.env.TEST_RUN_ID}-atomic-project-${id}`;
 
+  const getFlagsFromOptions = (options: BuildAppOptions) =>
+    options.pageId ? ['--pageId', options.pageId] : undefined;
+
+  const selectDefaultWithCli = async (buildTerminal: Terminal) => {
+    buildTerminal
+      .when(/Use an existing hosted search page/)
+      .on('stdout')
+      .do(answerPrompt(EOL))
+      .once();
+  };
+
+  const selectPageWithCli = (
+    options: BuildAppOptions,
+    buildTerminal: Terminal
+  ) => {
+    const osSpecificSelector = process.platform === 'win32' ? '>' : '\u276f'; //❯
+    const anotherPageSelectedMatcher = new RegExp(
+      `${osSpecificSelector} (?!${options.pageName})`,
+      'gm'
+    );
+    const expectedPageSelectedMatcher = new RegExp(
+      `${osSpecificSelector} ${options.pageName}`,
+      'gm'
+    );
+    buildTerminal
+      .when(anotherPageSelectedMatcher)
+      .on('stdout')
+      .do(answerPrompt('\x1B[B'))
+      .until(expectedPageSelectedMatcher);
+    buildTerminal
+      .when(expectedPageSelectedMatcher)
```

Comment:
A second use of said regexp: it was never triggered because the `until` above removed it first, because it was the first attached, and FIFO apply to listener [^1]

[^1]: https://nodejs.org/api/events.html#asynchronous-vs-synchronous